motion_model: 1 # Do not change this value

dt: 0.1

cov_xx: 0.0001
cov_xy: 0.00
cov_yy: 0.0001

survival_prob: 1.0
survival_dist: 0.0

birth_rate: 0.0

origin_x: 0.0
origin_y: 0.0
width: 61.515
height: 55.597

